TypeShark2/Client/Pages/GameMenuComponent.razor

```diff
@@ -0,0 +1,35 @@
+﻿<h3>Single Player</h3>
+<a href="/game" class="btn btn-primary">Let Me At 'Em</a>
+
+<h3 class="pt-4">Multi Player</h3>
+<button type="button" class="btn btn-primary" @onclick="SelectMultiPlayer">Let's Hunt Sharks, Together</button>
+
+<h3 class="pt-4">Game Icon</h3>
+@foreach (var item in Enum.GetNames(typeof(Data.GameIcon)))
+{
+    <div>
+        <input type="radio" name="icon" id="@item" value="@item"
+               @onchange="RadioSelection"
+               checked=@(Context.GameIcon.ToString().Equals(item, StringComparison.OrdinalIgnoreCase)) />
+        <label for="@item">@item</label>
+    </div>
+}
```

Comment:
I appreciate the radio UI exploration for a demo, but I'm not a fan of bringing this picker into the game and into the main branch.  Also, this is unrelated to upgrading to .Net 5.  Generally I prefer multiple PR's that are short and to the point rather than single larger ones.

That said I actually like the GameIcon idea.  It could be a useful way to make the game more interesting at different levels, like have a 🐟 if you're in levels 1-2, or throw in lots of single letter symbols via 🐠 for variety.  But again, better in a different PR.

Reply:
I've removed the GameIcon improvement for now. Like your twist to it, maybe in a future version it can be added. Maybe .NET 6.0.